controller/src/main/java/io/pravega/controller/server/AuthResourceRepresentation.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server;
+
+import io.pravega.common.Exceptions;
+
+/**
+ * A utility class with methods for preparing string representations of auth-protected resources.
+ */
+public class AuthResourceRepresentation {
+
+    public static String ofScopes() {
+        return "/";
+    }
+
+    public static String ofAScope(String scopeName) {
+        Exceptions.checkNotNullOrEmpty(scopeName, "scopeName");
+        return scopeName;
+    }
+
+    public static String ofStreams(String scopeName) {
```

Comment:
Given that these method's signatures don't differentiate by type streams from scopes. It might be helpful to make their names even more explicit. IE: "ofStreamsInScope" / "ofReaderGroupsInScope"

Reply:
Sure, that sounds great to me. Have pushed a commit with this change.